bellatrix.android/src/main/java/solutions/bellatrix/android/components/AndroidComponent.java

```diff
@@ -37,6 +37,7 @@
 import solutions.bellatrix.core.utilities.InstanceFactory;
 import solutions.bellatrix.core.utilities.Log;
 
+import java.awt.Dimension;
```

Comment:
What was wrong with the built-in Point?

Reply:
I made changes to the Layout module to use java.awt.Dimension and Point, so the module could be used by Playwright as well. Because of the changes in Layout module, there are changes in Mobile, Web, and Desktop modules.